app/routes/your-claims/view-claim.js

```diff
@@ -33,23 +33,30 @@ module.exports = function (router) {
             forEdit: forEdit(claimDetails.claim.Status),
             viewClaim: true,
             claimStatusHelper: claimStatusHelper,
-            claimEventHelper: claimEventHelper
+            claimEventHelper: claimEventHelper,
+            bankDetails: {},
+            isRequestInfoPayment: claimDetails.claim.Status === 'REQUEST-INFO-PAYMENT'
           })
       })
   })
 
   router.post('/your-claims/:dob/:reference/:claimId', function (req, res, next) {
```

Comment:
on validation error the bank account/sortcode are not supplied to view for display